app/models/concerns/attachmentable.rb

```diff
@@ -69,7 +69,7 @@ def appropriate_extension(attachment)
     original_extension       = Paperclip::Interpolations.extension(attachment, :original)
     proper_extension         = extensions_for_mime_type.first.to_s
     extension                = extensions_for_mime_type.include?(original_extension) ? original_extension : proper_extension
-    extension                = 'jpeg' if ['jpe', 'jfif'].include?(extension)
```

Comment:
Are we handling `jfif` elsewhere? This would cause an issue with vips

Reply:
Oops, unintentional change.